src/database/migrations/20191029135947-notification.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Notifications', {
+      id: { allowNull: false, autoIncrement: true, primaryKey: true, type: Sequelize.INTEGER },
+      modelName: { type: Sequelize.STRING, allowNull: false },
+      modelId: { type: Sequelize.INTEGER, allowNull: false },
+      type: { type: Sequelize.STRING, allowNull: false },
+      status: { type: Sequelize.STRING, allowNull: false, defaultValue: 'Unread' },
```

Comment:
let's change this to a Boolean datatype. The column name can be `isRead`. It should also default to `true`